packages/runner/README.md

```diff
@@ -0,0 +1,68 @@
+# @secbox/runner
+
+Top level @secbox module for usage in test frameworks.
+
+Run scanning for vulnerabilities just from your unit tests on CI phase.
+
+## Setup
+
+```bash
+npm i -s @secbox/runner
+```
+
+## Usage
+
+For access to API you need to obtain a NeuraLegion token (either personal or organization API key).
+Find out how to obtain [personal](https://docs.brightsec.com/docs/manage-your-personal-account#manage-your-personal-api-keys-authentication-tokens) and [organization](https://docs.brightsec.com/docs/manage-your-organization#manage-organization-apicli-authentication-tokens) API keys in the [NeuraLegion knowledgebase](https://docs.brightsec.com).
+
+By default, it will be read from `BRIGHT_TOKEN` environment variable,
+but you can also pass it as part of `ConfigurationOptions`. Please refer to `@secbox/core` package [documentation](https://github.com/NeuraLegion/secbox-sdk-js/tree/master/packages/core#credentials) for more details.
+
+```ts
```

Comment:
Let's describe step-by-step guide of how to configure and manage `SecRunner` and `SecScan`

Reply:
please check